test/integration/api/uploader/uploader_spec.js

```diff
@@ -1038,6 +1030,40 @@ describe("uploader", function () {
       });
     });
   });
+  describe(":ocr", function () {
+    const ocrType = "adv_ocr";
+
+    it("should support requesting ocr when uploading", async function () {
+      // Upload an image and request ocr details in the response
+      const result = await UPLOADER_V2.upload(IMAGE_FILE, {ocr: ocrType, tags: [TEST_TAG]});
+
+      // Ensure result includes properly structured ocr details
+      expect(result).not.to.be.empty();
+      expect(result.info).to.be.an("object");
+      expect(result.info.ocr).to.be.an("object");
+      expect(result.info.ocr).to.have.key(ocrType);
+      expect(result.info.ocr[ocrType]).to.have.key("status");
+      expect(result.info.ocr[ocrType]).to.have.key("data");
+    });
+
+    it("should support ocr parameter in explicit", async function () {
+      // Upload an image
+      const uploadResult = await UPLOADER_V2.upload(IMAGE_FILE, {
+        tags: [TEST_TAG]
+      });
+
+      // Call explicit on the uploaded image with ocr parameter
+      const explicitResult = await UPLOADER_V2.explicit(uploadResult.public_id, {
+        ocr: ocrType,
+        "tags": [TEST_TAG],
+        type: "upload"
+      });
+
+      // Ensure result isn't an error
```

Comment:
Does this test just ensures that nothing explodes? 
In theory we could pass any number of arguments other than OCR and this test would still work, right/

Reply:
From our experimenting, `explicit()` does not return the details of the ocr, ocrType, etc. (unlike the `upload()` api call which does [return it](https://github.com/cloudinary/cloudinary_npm/blob/e3e71a0a2a38da3f002e87c26a17b08130500513/test/integration/api/uploader/uploader_spec.js#L1043))

So all we can do is make sure the `explicit` call does not return an error.

Is there something else that can be done here?